README.md

```diff
@@ -1,141 +1,207 @@
+## Developing and testing with Docker ##
 
-**_Starting Development Server_**
-```bash
-$ make dev-start
+The fastest way to get started developing and testing will be to use
+[Docker](https://www.docker.com/). The following will build a development
+server with live-reload enabled and expose it at `http://localhost:4200`. Any
+code changes made by you in the `ncats-webui/src` directory will cause the site
+to automatically update in your browser. Run the following command from the
+root of the repository.
+
+### Starting the development server ###
+
+```console
+make dev-start
 ```
 
-**_Viewing Development Server Logs_**
-```bash
-$ make dev-logs
+### Viewing the development server logs ###
+
+```console
+make dev-logs
 ```
 
+### Stopping the development server ###
+
 When you are finished developing stop the development server.
 
-**_Stopping Development Server_**
-```bash
-$ make dev-stop
+```console
+make dev-stop
 ```
 
-If you are finished with the project, you can clean up all volumes and images with the following:
+### Cleaning up ###
 
-**_Cleaning Up_**
-```bash
-$ make dev-clean
+If you are finished with the project, you can clean up all volumes and images
+with the following:
+
+```console
+make dev-clean
 ```
 
+## Adding components, services, etc. with Angular CLI ##
 
-##### Adding Angluar-CLI Components, Services, etc...
-If you want to add new web pages, services, etc, ng is the tool to do the job. It is the angular-cli tool, which is what this project is built on.
+If you want to add new web pages, services, etc., ng is the tool to do the job.
+It is the Angular CLI tool, which is what this project is built on.
 
-**_Executing ng commands_**
-```bash
+### Executing ng commands ###
+
+```console
 $ make dev-shell
 root@container# ng g component my-component
 root@container# ng g service my-service
 ```
 
-Ensure that the addition is added to the `app.module.ts` file with the rest of it's type.
+Ensure that the addition is added to the `app.module.ts` file with the rest of
+it's type.
 
-For more information on the usage of ng, visit [https://github.com/angular/angular-cli/](https://github.com/angular/angular-cli/)
+For more information on the usage of ng, visit
+[https://github.com/angular/angular-cli/](https://github.com/angular/angular-cli/)
 
-##### Adding Packages
-Since the npm/bower package installation occurs in the building of the image, you will need to add your package using whatever method you like (easiest is npm install --save).
+## Adding Packages ##
 
-**_Install NPM or Bower Packages_**
-```bash
+Since the npm/bower package installation occurs in the building of the image,
+you will need to add your package using whatever method you like (easiest is
+  npm install --save).
```

Comment:
The version of `npm` for this configuration is so old you need to use `--save` to have the package be added as a dependency. Are you ok with:

```suggestion
`npm install --save`, as shown below).
```

Reply:
Yes.  I originally thought that it should read `npm install <package_name> --save`, but since we are spelling it all out a few lines later, I shortened it to just "`npm install`, as shown below".  I am fine with your suggestion though.